scipy/special/_orthogonal.py

```diff
@@ -389,15 +389,22 @@ def wfunc(x):
 
 
 def roots_sh_jacobi(n, p1, q1, mu=False):
-    """Gauss-Jacobi (shifted) quadrature.
+    r"""Gauss-Jacobi (shifted) quadrature.
 
     Compute the sample points and weights for Gauss-Jacobi (shifted)
     quadrature. The sample points are the roots of the nth degree
-    shifted Jacobi polynomial, :math:`G^{p,q}_n(x)`. These sample
-    points and weights correctly integrate polynomials of degree
-    :math:`2n - 1` or less over the interval :math:`[0, 1]` with
+    shifted Jacobi polynomial
+
+    .. math::
+
+        G_n^{(p, q)}(x)
+          = \binom{2n + p - 1}{n}^{-1}P_n^{(p - q, q - 1)}(2x - 1).
```

Comment:
Since `p` and `q` are reals, it might be a bit confusing to use the binomial coefficient here.  Some equivalent alternatives that are defined for real `p` are

- `factorial(n) * gamma(n + p) / gamma(2 * n + p)`  (`gamma` = euler gamma function)
- `factorial(n) / poch(n + p, n)`  (`poch` = pochhammer symbol for the rising factorial)
- `n * beta(n, n + p)` (`beta` = euler beta function)

The pochhammer symbol might be a nice fit with the underlying hypergeometric function.
Either way, no need to explain what any of these three special functions are, because you could just link to the docs of their respective `scipy.special` function.

But that being said, this definition is the same as the one used in `eval_sh_jacobi`. So I can also see how you might consider this out-of-scope.

Reply:
Concerning AS vs DLMF, if replacing AS references by DLMF references is considered worth doing, I might work on that once the present issue is closed. While this task will ultimately concern the whole special functions module, it probably makes sense to first concentrate on the functions related to orthogonal polynomials. In my experience, the chapter on orthogonal polynomials has changed its structure more than other chapters in the transition from AS to DLMF.